LibraBFT/Impl/Consensus/EpochManager.agda

```diff
@@ -423,6 +440,14 @@ start self0 now obmPayload obmNeedFetch obmProposalGenerator obmLedgerInfoWithSi
   - in other cases it sends messages (via 'stps') to other nodes
 
 The following "implementation" compiles, but has lots of TEMPORARY things to get it to compile.
+In the proofs, the System Model "handles" input and output.
+Therefore 'obmStartLoop' will not be implemented nor "proved".
+It is here to show what is done in the Haskell code.
+If the Agda code were ever to be extracted to executable Haskell code,
+then this code would be completed.
+
+The system model will need to prove things about 'EpochManager.processMessage'.
```

Comment:
This comment doesn't make sense.  The system model doesn't even know about `EpochManager.processMessage`, let alone prove anything about it.  I /think/ what you mean is that the system model will be instantiated with (something like) 'EpochManager.processMessage' and that will be the system about which we prove our properties.  Maybe?

Reply:
Yes - "instantiate".  DONE